charts/bindplane/templates/bindplane.yaml

```diff
@@ -146,6 +146,48 @@ spec:
               value: /credentials.json
             {{- end }}
             {{- end }}
+            {{- if and (eq .Values.enterprise true) (eq .Values.eventbus.type "kafka") }}
+            - name:  BINDPLANE_KAFKA_BROKERS
+              value: {{ .Values.eventbus.kafka.brokers }}
+            - name:  BINDPLANE_KAFKA_VERSION
+              value: "{{ .Values.eventbus.kafka.version }}"
+            - name:  BINDPLANE_KAFKA_TOPIC_NAME
```

Comment:
I changed this after I realized you couldn't init the topic. Now it's just `BINDPLANE_KAFKA_TOPIC`